test/visual-diff/visual-tests

```diff
@@ -19,15 +19,13 @@
  */
  {
   /**
-   * Path relative to amphtml/ that contains the assets for all test pages.
-   */
-  "assets_dir": "examples/visual-tests",
-
-  /**
-   * Path relative to webserver root where assets for all test pages are
-   * served.
-   */
-  "assets_base_url": "/examples/visual-tests",
+  * An array of glob strings to use to load assets to Percy, relative to
+  * the base amphtml/ directory.
+  * See: glob primer https://github.com/isaacs/node-glob#glob-primer
+  */
+  "asset_globs": [
```

Comment:
Aha, I now see how this works. Might be worth documenting the fact that this array will have to be updated each time someone adds a new test.